src/components/crossDevice/ClientIntro/index.tsx

```diff
@@ -12,11 +12,21 @@ import style from './style.scss'
 import type { StepComponentBaseProps } from '~types/routers'
 
 type Props = {
+  crossDeviceClientIntroProductName?: string
+  crossDeviceClientIntroProductLogoSrc?: string
   nextStep: () => void
 } & StepComponentBaseProps
 
-const CrossDeviceClientIntro: FunctionComponent<Props> = ({ nextStep }) => {
+const CrossDeviceClientIntro: FunctionComponent<Props> = ({
+  nextStep,
+  crossDeviceClientIntroProductName,
+  crossDeviceClientIntroProductLogoSrc,
+}) => {
   const { translate } = useLocales()
+  const defaultSubtitle = translate('cross_device_session_linked.subtitle')
+  const subtitle = crossDeviceClientIntroProductName
+    ? `${defaultSubtitle} ${crossDeviceClientIntroProductName}`
```

Comment:
String template in this situation is a bit hard to follow. I'd suggest something more traditional:
```suggestion
    ? `[defaultSubtitle, crossDeviceClientIntroProductName].join(' ')`
```

Reply:
No need for `` in the above suggestion